sandbox-backend/src/main/resources/db/migration/V0000029__addAddressToCompany.sql

```diff
@@ -0,0 +1,15 @@
+DROP TABLE company_country;
```

Comment:
We already have version 29 in the project (_V0000029__addEmails_Subscriptions.sql_). Please update the version.

Reply:
I changed the version.